src/jobManagerClient.ts

```diff
@@ -333,6 +333,25 @@ export class JobManagerClient extends HttpClient {
     }
   }
 
+  public async createTaskForJob<T>(jobId: string, taskBody: ICreateTaskBody<T>): Promise<void> {
+    const createTaskForJobUrl = `/jobs/${jobId}/tasks`;
+    try {
+      this.logger.debug({ jobId, msg: `createTaskForJob of task with jobId=${jobId}` });
```

Comment:
add the following details to log: {
        url: createTaskForJobUrl,
        targetService: this.targetService,
        jobId,
        msg: `createTaskForJob jobId=${jobId}`,
      }

Reply:
should be info

Reply:
you can add debug log with 'taskBody'